Source/App/EbAppMain.c

```diff
@@ -209,16 +217,20 @@ int main(int argc, char* argv[])
                                 exitConditionsInput[instanceCount] = ProcessInputBuffer(
                                                                             configs[instanceCount],
                                                                             appCallbacks[instanceCount]);
+                            if (exitConditionsRecon[instanceCount] == APP_ExitConditionNone)
+                                exitConditionsRecon[instanceCount] = ProcessOutputReconBuffer(
+                                                                            configs[instanceCount],
+                                                                            appCallbacks[instanceCount]);
                             if (exitConditionsOutput[instanceCount] == APP_ExitConditionNone)
                                 exitConditionsOutput[instanceCount] = ProcessOutputStreamBuffer(
                                                                             configs[instanceCount],
                                                                             appCallbacks[instanceCount],
-                                                                            exitConditionsInput[instanceCount] == APP_ExitConditionNone ? 0 : 1);
-                            if ((exitConditionsOutput[instanceCount] != APP_ExitConditionNone && exitConditionsInput[instanceCount] != APP_ExitConditionNone)||
-                                (exitConditionsOutput[instanceCount] == APP_ExitConditionError || exitConditionsInput[instanceCount] == APP_ExitConditionError)){
+                                                                            (exitConditionsInput[instanceCount] == APP_ExitConditionNone) || (exitConditionsRecon[instanceCount] == APP_ExitConditionNone)? 0 : 1);
+                            if ((exitConditionsRecon[instanceCount] != APP_ExitConditionNone && exitConditionsOutput[instanceCount] != APP_ExitConditionNone && exitConditionsInput[instanceCount] != APP_ExitConditionNone)||
+                                (exitConditionsRecon[instanceCount] == APP_ExitConditionError || exitConditionsOutput[instanceCount] == APP_ExitConditionError || exitConditionsInput[instanceCount] == APP_ExitConditionError)){
                                 channelActive[instanceCount] = EB_FALSE;
```

Comment:
Hi, logic error here.
If recon is disabled.  A normal encoding after 1 loop will be set to channelActive[instanceCount] = EB_FALSE.
In this if(), I don't think you need to consider exitConditionsRecon, since it just for debugging purpose?